tutorials/README.md

```diff
@@ -2,14 +2,11 @@
 
 In this folder you can find a collection of useful tutorials in order to understand the principles and the potential of **EZyRB**.
 
-#### [Tutorial 1 - Offline phase](tutorial-1-offline-phase.ipynb)
-Here we show the typical workflow for the construcion of the reduced basis space, called offline phase. The model reduction is done on a vtk pressure field.
-
-#### [Tutorial 2 - Online phase](tutorial-2-online-phase.ipynb)
-In this tutorial we show the workflow for the online evaluation of the output of interest. It is the natural continuation of the first tutorial on the vtk pressure field.
-
-#### [Tutorial 3 - Output interpolation](tutorial-3-interpolation.ipynb)
-Here we show how use the module mapper in order to interpolate the solutions (computed on meshes with different dimensions) on a new fixed dimension mesh.
+#### [Tutorial 1](tutorial-1.ipynb)
+Here we show the typical workflow on a vtk pressure field for:
+1. The construcion of the reduced basis space, called offline phase;
+2. The online evaluation of the output of interest;
+3. Using the module mapper in order to interpolate the solutions (computed on meshes with different dimensions) on a new fixed dimension mesh.
```

Comment:
This module does not exist anymore

Reply:
I didn't go through the details of the new tutorial, do you mean the _module mapper_ mentioned in the third point?

Reply:
Yes, the module _mapper_ does not exist since the new version of EZyRB, mainly because the file handling part has been removed from the package (it will be incorporated in a third package, hopefully very soon).
Please remove the third bullet.